src/bumper/bumper.go

```diff
@@ -16,16 +16,41 @@ var ErrNoTags = errors.New("source did not return any tag")
 // Bumper takes a changelog and a version and figures out the next one.
 type Bumper struct {
 	changelog     changelog.Changelog
-	EntryCap      bump.Type
-	DependencyCap bump.Type
+	entryCap      bump.Type
+	dependencyCap bump.Type
 }
 
+// BumperOption is the interface to add options to the bumper.
+//
+//nolint:revive // Following the options pattern here. Renaming it to `option` could lead to a misunderstanding.
+type BumperOption func(bumper *Bumper)
+
 // New creates a new bumper.
-func New(c changelog.Changelog) Bumper {
-	return Bumper{
+func New(c changelog.Changelog, opts ...BumperOption) Bumper {
```

Comment:
I like the options pattern as much as anyone, but in this case, where the only options we can have in the foreseeable future are just plain values... Should we go with simplicity instead and just export the fields?

Reply:
As we are not creating a public API I think it does not make sense to have a strong encapsulation. Also improves readability.

Removed in [#8c22e2d](https://github.com/newrelic/release-toolkit/pull/128/commits/8c22e2d1ddfd7549a343c8de3996a1d153d4d202).